api/src/routers/future-meals.js

```diff
@@ -0,0 +1,11 @@
+import express from "express";
+import knex from "../database_client.js";
+
+
+const futureMeal = express.Router();
+const currectDay=new Date();
```

Comment:
We have a bug 🐛 here, the date is declared in the [global ](https://developer.mozilla.org/en-US/docs/Glossary/Global_scope) scope, and it will not be updated every time the user accesses your API.

This means that if you run your server today and someone accesses the future meals API in a week, they will get the meals from the point in time when you started the server.

You can create two APIs in your index.js as defined below to understand the difference:

```js
apiRouter.get("/now-local-scope", (req, res) => {
    const nowLocal = new Date();
    res.json({ time: nowLocal });
});

const nowGlobal = new Date();
apiRouter.get("/now-global-scope", (req, res) => {
    res.json({ time: nowGlobal});
});
```

You're going to notice that the global one does not change